cloudbuild/nightly/scripts/unbounded_table_write.sh

```diff
@@ -19,37 +19,49 @@ UNBOUNDED_JOB_SINK_PARALLELISM=$3
+# Copy Source File to a temp directory
+gcloud storage cp "$GCS_SOURCE_URI"source.csv "$NEW_GCS_SOURCE_URI"source.csv
+# Lifecycle policy of deletion in 1 day already set, no need to add expiration to this directory
 
 # Modify the destination table name for all tests.
-DESTINATION_TABLE_NAME="$SOURCE_TABLE_NAME"-sink
-# Create the destination table from the source table schema.
-python3 cloudbuild/nightly/scripts/python-scripts/create_sink_table.py -- --project_name "$PROJECT_NAME" --dataset_name "$DATASET_NAME" --source_table_name "$SOURCE_TABLE_NAME" --destination_table_name "$DESTINATION_TABLE_NAME"
+DESTINATION_TABLE_NAME="$DESTINATION_TABLE_NAME"_"$timestamp"
+
+if [ "$IS_SQL" == True ]
+then
+  echo "SQL Mode is Enabled!"
+  DESTINATION_TABLE_NAME="$DESTINATION_TABLE_NAME"_SQL
+fi
+if [ "$IS_EXACTLY_ONCE_ENABLED" == True ]
+then
+  DESTINATION_TABLE_NAME="$DESTINATION_TABLE_NAME"_EXO
+else
+  DESTINATION_TABLE_NAME="$DESTINATION_TABLE_NAME"_ALO
+fi
+# Create the destination table from hardcoded table schema.
+python3 cloudbuild/nightly/scripts/python-scripts/create_unbounded_sink_table.py -- --project_name "$PROJECT_NAME" --dataset_name "$DATASET_NAME" --destination_table_name "$DESTINATION_TABLE_NAME"
 # Set the expiration time to 1 hour.
 bq update --expiration 3600 "$DATASET_NAME"."$DESTINATION_TABLE_NAME"
 
 # Running this job async to make sure it exits so that dynamic data can be added
-gcloud dataproc jobs submit flink --id "$JOB_ID" --jar="$GCS_JAR_LOCATION" --cluster="$CLUSTER_NAME" --region="$REGION" --properties="$PROPERTIES" --async -- --gcp-source-project "$PROJECT_NAME" --bq-source-dataset "$DATASET_NAME" --bq-source-table "$SOURCE_TABLE_NAME" --mode unbounded --ts-prop "$TS_PROP_NAME" --partition-discovery-interval "$PARTITION_DISCOVERY_INTERVAL" --gcp-dest-project "$PROJECT_NAME" --bq-dest-dataset "$DATASET_NAME" --bq-dest-table "$DESTINATION_TABLE_NAME" --sink-parallelism "$UNBOUNDED_JOB_SINK_PARALLELISM" --is-sql "$IS_SQL" --exactly-once "$IS_EXACTLY_ONCE_ENABLED"
+gcloud dataproc jobs submit flink --id "$JOB_ID" --jar="$GCS_JAR_LOCATION" --cluster="$CLUSTER_NAME" --region="$REGION" --properties="$PROPERTIES" --async -- --gcp-source-project "$PROJECT_NAME" --gcs-source-uri "$NEW_GCS_SOURCE_URI" --mode unbounded --file-discovery-interval "$FILE_DISCOVERY_INTERVAL" --gcp-dest-project "$PROJECT_NAME" --bq-dest-dataset "$DATASET_NAME" --bq-dest-table "$DESTINATION_TABLE_NAME" --sink-parallelism "$UNBOUNDED_JOB_SINK_PARALLELISM" --is-sql "$IS_SQL" --exactly-once "$IS_EXACTLY_ONCE_ENABLED"
 
-# Dynamically adding the data. This is timed 2.5 min wait for read and 5 min refresh time.
-python3 cloudbuild/nightly/scripts/python-scripts/insert_dynamic_partitions.py -- --project_name "$PROJECT_NAME" --dataset_name "$DATASET_NAME" --table_name "$SOURCE_TABLE_NAME" --refresh_interval "$PARTITION_DISCOVERY_INTERVAL" --is_write_test
+# Dynamically adding new files. This is timed 2.5 min wait for read and 5 min refresh time.
```

Comment:
Let's remove the `--is_write_test` for unbounded and correspondingly in the python file as well.

Reply:
Removed